tsconfig.npm-package.json

```diff
@@ -1,7 +1,10 @@
 {
   "extends": "./tsconfig",
   "compilerOptions": {
-    "declarationDir": "dist",
+    // TODO define declarationDir in npm script for clarity, this also to remove the tsconfig.typedoc.json file (use this file instead)
```

Comment:
❓ There is some `TODO`. Is it normal ?

Reply:
no, I missed it 😄

Reply:
fixed in [`8fd7185` (#2325)](https://github.com/process-analytics/bpmn-visualization-js/pull/2325/commits/8fd71856a631dee669f427dae2fee8b987c99106)